packages/contracts/package.json

```
@@ -55,8 +55,8 @@
     "@nomiclabs/hardhat-ethers": "^2.0.1",
     "@nomiclabs/hardhat-etherscan": "^2.1.6",
     "@nomiclabs/hardhat-waffle": "^2.0.1",
-    "@openzeppelin/contracts": "^3.3.0",
-    "@openzeppelin/contracts-upgradeable": "^3.3.0",
+    "@openzeppelin/contracts": "4.3.1",
+    "@openzeppelin/contracts-upgradeable": "4.3.1",
```

```suggestion
    "@openzeppelin/contracts": "4.3.2",
    "@openzeppelin/contracts-upgradeable": "4.3.2",
```

[4.3.2 includes a security patch](https://github.com/OpenZeppelin/openzeppelin-contracts/releases/tag/v4.3.2). Our codebase doesn't make use the contracts that were affected by the vuln, but it's good to do anyways.